contracts/address-manager/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Build results
```

Comment:
Do we need files like this if they exist in the root of this repo?

Reply:
Good point. Probably not because this is a workspace and build output ends up in the root. Removing..